adding Temporalite to docs (#1414)
temporalio/documentation: docs/clusters/quick-install.md

---
id: quick-install
title: How to quickly install a Temporal Cluster
sidebar_label: Install Temporal Cluster
description: Quickly install and run a Temporal Cluster in various ways.
---

> The following section list different methods of deploying your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally, so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl) commands to test and develop applications.

Reviewer: ```suggestion
The following sections list various methods of deploying your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally, so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl) commands to test and develop applications.
```

- [Temporalite](#temporalite): This distribution of Temporal runs as a single process with zero runtime dependencies.

> - [Docker](#docker-compose): Using Docker Compose makes it easy to develop your Temporal Application.

Reviewer: ```suggestion
- [Docker](#docker-compose): Using Docker Compose simplifies developing your Temporal Application.
```
A claim that something is "easy" is subject to wide interpretation. See https://developers.google.com/style/word-list?hl=en#easy.

- [Gitpod](#gitpod): One-click deployments are available for Go and TypeScript.

> **For information on deploying a [production environment](/server/production-deployment), see the [Temporal Cloud](../cloud/) documentation.**

Reviewer: ```suggestion
**For information on deploying a [production environment](/server/production-deployment), see the [Temporal Cloud](/cloud) documentation.**
```

#### Temporalite

Temporalite is a distribution of Temporal that runs as a single process with zero runtime dependencies.
It supports persistence to disk and in-memory mode through SQLite.

> **Prerequisites**

Reviewer: ```suggestion
**Prerequisites**

```

Temporalite requires Go 1.18 or later.

**Build and start Temporalite**

The following steps start and run a Temporal Cluster.

1. Build from source by using `go install`.
   ```bash
   go install github.com/temporalio/temporalite/cmd/temporalite@latest
   ```
2. Start Temporalite by using the `start` command.
   ```bash
   temporalite start --namespace default
   ```

>    Replace `default` with your [Namespace Name](../concepts/what-is-a-cloud-namespace-name/).

Reviewer: ```suggestion
   Replace `default` with your [Namespace Name](/cloud/#temporal-cloud-namespace-name).
```

**Results**: You should have Temporal Cluster running at `http://127.0.0.1:7233` and the Temporal Web UI at [`http://127.0.0.1:8233`](http://127.0.0.1:8233/namespaces/default/workflows).

<!-- For macOS users, if you receive the `error setting up schema: stat /Users/<user_name>/Library/Application Support/temporalite/db:` error, then create the folders `temporalite/db` in your `Application Support` library. -->

#### Docker Compose

Use Docker Compose and Temporal Cluster Docker images to quickly install and run a Temporal Cluster locally while developing Temporal Applications.

> **Prerequisites**

Reviewer: ```suggestion
**Prerequisites**

```

Install [Docker](https://docs.docker.com/engine/install) and [Docker Compose](https://docs.docker.com/compose/install).

> **Clone the repo and run Docker Compose**

Reviewer: ```suggestion
**Clone the repo and run Docker Compose**

```

The following steps start and run a Temporal Cluster using the default configuration.

1. Clone the [temporalio/docker-compose](https://github.com/temporalio/docker-compose) repository.
   ```bash
   git clone https://github.com/temporalio/docker-compose.git
   ```
2. Change to the directory for the project.
   ```bash
   cd docker-compose
   ```

> 3. From your project directory, start up your application by running docker compose up.

Reviewer: ```suggestion
3. From your project directory, start your application.
```

> It can take up to a full minute for the one-click deployments to get fully up and running.

Reviewer: ```suggestion
A one-click deployment can take up to a full minute to get fully up and running.
```
